### ISSUE TITLE
[Bug]: hcloud_load_balancer_service

### ISSUE THREAD

**Issue by yassin213** [hetznercloud/terraform-provider-hcloud#750]
### What happened?

Hi, hier i found Bug.

```tf
resource "hcloud_load_balancer" "load_balancer" {
  name               = "my-load-balancer"
  load_balancer_type = "lb11"
  location           = "nbg1"
}

resource "hcloud_load_balancer_service" "load_balancer_service" {
    load_balancer_id = hcloud_load_balancer.test_load_balancer.id
    protocol         = "http"
} 
```

by hcloud_load_balancer_service creation it shoul call the hcloud_load_balancer Name **my-load-balancer** and not **test_load_balancer** please correct it!

the URL: https://registry.terraform.io/providers/hetznercloud/hcloud/latest/docs/resources/load_balancer_service

the correct code:

```tf
resource "hcloud_load_balancer_service" "load_balancer_service" {
    load_balancer_id = hcloud_load_balancer.my-load-balancer.id
    protocol         = "http"
}
```
Best Regards!

### What did you expect to happen?

the correct code:
```tf
resource "hcloud_load_balancer_service" "load_balancer_service" {
    load_balancer_id = hcloud_load_balancer.my-load-balancer.id
    protocol         = "http"
}
```

### Please provide a minimal working example

the correct code:

```tf
resource "hcloud_load_balancer_service" "load_balancer_service" {
    load_balancer_id = hcloud_load_balancer.my-load-balancer.id
    protocol         = "http"
}
```

**Comment by apricote** [comment]
Hey @yassin213, 

thanks for noticing this! The example is definitely wrong.

You do need to use the terraform resource name (`load_balancer`) though in the references and not the `name` attribute that will be used in the Hetzner Cloud resource.

I think this is the correct code:

```tf
resource "hcloud_load_balancer" "load_balancer" {
  name               = "my-load-balancer"
  load_balancer_type = "lb11"
  location           = "nbg1"
}

resource "hcloud_load_balancer_service" "load_balancer_service" {
    load_balancer_id = hcloud_load_balancer.load_balancer.id
    protocol         = "http"
}
```

Do you want to open a Pull Request? The source for the website is [here](https://github.com/hetznercloud/terraform-provider-hcloud/blob/f72cd3bdac6acc1b80bc20ffdf766f52a3385d2d/website/docs/r/load_balancer_service.html.md?plain=1#L16-L25)

**Comment by apricote** [comment]
Fixed in #760
